### DISCUSSION TITLE
Maui with Razor class library layout

### DISCUSSION THREAD

**Discussion by inusitado** [CommunityToolkit/Maui#1838] [General]
"In my hybrid Maui Blazor project with a Razor class library, when I include CommunityToolkit.Maui, the layout stops working. If I downgrade CommunityToolkit.Maui to version 6.0, the layout works again. Any suggestions?"

**Comment by CliffAgius** [comment]
Did you manage to resolve this issue? I am going through old discussions and just seen this and wondered if you still need help as it's so old or if we can close this.
